include/net/net_context.h

```diff
@@ -27,7 +27,6 @@
 #include <net/net_if.h>
 #include <net/net_stats.h>
 
-#include <can.h>
```

Comment:
You should not add commit that fixes your previous commit. 

just fixup the original commit. 
git rebase -i HEAD~2

there you'll see your 2 commits, in front of that one "net: corrected indentation"  replace "pick" by "f", save/exit and that should be done.

Then git -f push  (-f for forcing the update)

And that's it.